packages/runtime/src/templates/edge/runtime.ts

```diff
@@ -82,7 +82,7 @@ const handler = async (req: Request, context: Context) => {
   const request: RequestData = {
     headers: Object.fromEntries(req.headers.entries()),
     geo,
-    url,
+    url: req.url,
```

Comment:
Was this broken before or were we doing `request.toString()` currently where this is being called?

Reply:
This is passed to Next.js. It broke things here: https://github.com/netlify/next-runtime/actions/runs/3321501353/jobs/5489233564#step:10:348